include/sentry.h

```diff
@@ -72,6 +72,8 @@ extern "C" {
 #ifndef SENTRY_SDK_NAME
 #    if defined(SENTRY_PLATFORM_ANDROID)
 #        define SENTRY_SDK_NAME "sentry.native.android"
+#    elif defined(SENTRY_PLATFORM_XBOX)
+#        define SENTRY_SDK_NAME "sentry.native.xbox"
```

Comment:
I didn't highlight this because it would be fine to do this in a closed repo. So, with the toolchain defs removed, no one could use the open _alone_ repo anymore anyway, so having the def here wouldn't provide a lot in return. However, I am fine either way. You guys delineate which parts should be where.

Reply:
I've been thinking along the same line but @bruno-garcia asked me to put this back because it's possible sometimes can still use sentry-native directly if they use their own toolchain file (e.g. if they pull this directly to their build system instead of precompiling.